stellar-contract-env-host/src/storage.rs

```diff
@@ -1,14 +1,11 @@
 use std::rc::Rc;
 
-use crate::xdr::{Hash, ScVal};
+use crate::xdr::{LedgerEntry, LedgerKey};
 use crate::HostError;
 use im_rc::OrdMap;
 
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
-pub struct Key {
-    pub contract_id: Hash,
-    pub key: ScVal,
-}
+pub type Key = LedgerKey;
```

Comment:
I think these typedefs kinda obscure more than they elucidate. I'd just put the LedgerKey / LedgerEntry types in the places they're used.

Reply:
Fixed.